js/Wallop.js

```diff
@@ -54,11 +54,16 @@
     this.createCustomEvent();
 
     // If there is no active item, start at 0
-    if (this.currentItemIndex < 0) {
+    if (this.currentItemIndex === -1) {
       this.currentItemIndex = 0;
       addClass(this.allItemsArray[this.currentItemIndex], this.options.currentItemClass);
     }
 
+    // Update the button state on initialization when the `carousel` option isn't enabled
+    if (!this.carousel) {
+      this.updateButtonStates();
```

Comment:
When the `carousel` option is disabled, the previous button gets disabled when you are on the first slide and the next button gets disabled when you are on the last slide. When the wallop slider is initialized and the start item is the first or the last one, the depending button isn't disabled (see http://codepen.io/anon/pen/zvrWpp).